ftw/simplelayout/mapblock/contents/configure.zcml

```diff
@@ -0,0 +1,22 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    i18n_domain="ftw.simplelayout">
+
+    <!-- XXX Implement a specific KMLLayer for Simplelayout and do not recycle the one from geo.kml -->
```

Comment:
There's a adapter lookup somewhere in c.geo for a specific *geo* View marker interface. All geo viewlets are assigned to this view interface. BUT we don't want those viewlets on a block. So I had to define a separate adapter for block views.